terraform/vars/terraform-dev.tfvars

```diff
@@ -5,7 +5,7 @@ rw_api_url                     = "https://api.resourcewatch.org"
 desired_count                  = 1
 auto_scaling_min_capacity      = 1
 auto_scaling_max_capacity      = 5
-lambda_analysis_workspace      = "features-lat_lon"
+lambda_analysis_workspace      = "feature-otf_lists"
```

Comment:
I assume this is irrelevant/ignored once we are running in develop/staging?

Reply:
yes